src/react-fontawesome.css

```diff
@@ -0,0 +1,10 @@
+.sr-only {
```

Comment:
I was thinking it was painful to have inline CSS. And it would be complicated to test for it I think? But I can do it if you want.